### ISSUE TITLE
COVID symptoms form

### ISSUE THREAD

**Issue by fatousylla** [medic/config-msf#157]
With the question "Is patient suspected of other pathology?" the condition is not respected. We have this question even though we respond by yes to the patient having COVID symptoms accordingly to the XLS. 
cc @mmureithi

**Comment by mmureithi** [comment]
No, that is a flaw. "Is patient suspected of other pathology?" should only be asked if COVID is ruled out. @kitsao kindly review the [logic](https://docs.google.com/spreadsheets/d/1Mz8lNOYxYk6AYqhjUazMdQZVCgOu6Lc7wQxh6D-5Igk/edit#gid=332183608) here to identify the issue highlighted.

**Comment by kitsao** [comment]
Effected.

**Comment by fatousylla** [comment]
done
